.mailmap

```diff
@@ -19,6 +19,7 @@ Abhishek Lekshmanan <abhishek@suse.com> <alekshmanan@suse.com>
 Adam C. Emerson <aemerson@redhat.com>
 Adam King <adking@redhat.com> <kingamk3@gmail.com>
 Adam King <adking@redhat.com> <adking@redhat,com>
+Adam King <adking@redhat.com> <47704447+adk3798@users.noreply.github.com>
```

Comment:
The `noreply.github.com` items  look weird, to be honest. I looked over `git log` and they came from merges performed by the github's web interface. 

```
commit d9030299efd7ddfca81c1a3048d56b655fe7dc81
Merge: 14355eb39ad 187baffd56f
Author: Adam King <47704447+adk3798@users.noreply.github.com>
Date:   Wed Apr 3 13:06:55 2024 -0400

    Merge pull request #56628 from gbregman/main
    
    mgr/cephadm: add some NVMEof gateway fields to be configurable by cephadm
    
    Reviewed-by: Adam King <adking@redhat.com>
```

I don't have any strong feelings here, just was curious.

Reply:
the goal of mappings like this is to make sure that all of Adam's contributions get credited to a single individual. this change prevents entries like `Adam King <47704447+adk3798@users.noreply.github.com>` from showing up separately in the contributor credits